packages/package-json-lint/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+## 1.6.0 - (July 10, 2023)
```

Comment:
This should be a major version bump (v2.0)

Reply:
Good catch! Thanks